symfit/tests/tests_ode.py

```diff
@@ -137,17 +137,19 @@ def test_single_eval(self):
 
         # The harmonic oscillator as a system, >1st order is not supported yet.
         harmonic_dict = {
-            D(x, t): - k * y,
-            D(y, t): k * x,
+            Derivative(x, t): - k * y,
+            Derivative(y, t): k * x,
         }
         harmonic_model = ODEModel(harmonic_dict, initial={t: 0.0, x: 1.0, y: 0.0})
 
         tdata = np.linspace(0, 100, 101)
         X, Y = harmonic_model(t=tdata, k=0.1)
         for index, t in enumerate(tdata):
             X_point, Y_point = harmonic_model(t=t, k=0.1)
-            self.assertAlmostEqual(X_point, X[index])
-            self.assertAlmostEqual(Y_point, Y[index])
+            np.testing.assert_allclose(X_point, X[index])
+            np.testing.assert_allclose(Y_point, Y[index])
```

Comment:
Why is this preferred? As it stands now `X_point, Y_point` are just floats right? Maybe you also mean to get rid of the for loop?